GitHubActions/build.sh

```diff
@@ -53,7 +53,7 @@ if [ "$MODE" = "example" -o "$MODE" = "all" ]; then
   gem install bundler
   bundle check || bundle install
   bundle exec pod install
-  set -o pipefail && xcodebuild build analyze -workspace ChatExample.xcworkspace -scheme ChatExample -destination "platform=iOS Simulator,name=iPhone 11 Pro" CODE_SIGNING_REQUIRED=NO | xcpretty -c
+  set -o pipefail && xcodebuild build analyze -workspace ChatExample.xcworkspace -scheme ChatExample -destination "platform=iOS Simulator,name=iPhone 11 Pro" CODE_SIGNING_REQUIRED=NO OTHER_CFLAGS="-Xclang -fcompatibility-qualified-id-block-type-checking" | xcpretty -c
```

Comment:
One of the dependencies `PINRemoteImage` is not compatible with Xcode 12.
The compiler complains that there's incompatible block pointers.

So I added workaround to disable this warning by passing extra flags.